src/test/scala/io/iohk/ethereum/Fixtures.scala

```diff
@@ -130,7 +130,7 @@ object Fixtures {
         transactionsRoot = ByteString(Hex.decode("56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421")),
         receiptsRoot = ByteString(Hex.decode("56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421")),
         logsBloom = ByteString(Hex.decode("0" * 512)),
-        difficulty = BigInt("17179869184"),
+        difficulty = BigInt("1"),
```

Comment:
how does this affect the tests? seems arbitrary and not related to how best block is stored?

Reply:
it's not related, this is again a change that I had to do to run a test. The genesis block difficulty was higher than the block that were being added to the blockchain(could have had just increase the blocks difficulty instead). I would say it's not important at the moment, and can change it back